fda-model/compute/LeveeMessage.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace compute
+{
+    public class LeveeMessage : Base.Interfaces.IMessage
```

Comment:
this is neat. Why do we need to pass out the top of levee? we should only do this if we need to pass more than a string, otherwise we should use the Base.Implementations.Message or one of the Model.Implementations.Messages which is sufficient for describing a string message.

Reply:
I didn't really know how to do it differently. Messaging and validation is still very unclear to me.

Reply:
we can work through it together.